x-pack/plugins/features/server/plugin.ts

```diff
@@ -93,19 +89,13 @@ export class FeaturesPlugin
     Plugin<RecursiveReadonly<PluginSetupContract>, RecursiveReadonly<PluginStartContract>> {
   private readonly logger: Logger;
   private readonly featureRegistry: FeatureRegistry = new FeatureRegistry();
-  private isTimelionEnabled: boolean = false;
   private isReportingEnabled: boolean = false;
 
   constructor(private readonly initializerContext: PluginInitializerContext) {
     this.logger = this.initializerContext.logger.get();
   }
 
-  public setup(
-    core: CoreSetup,
-    { visTypeTimelion }: { visTypeTimelion?: TimelionSetupContract }
```

Comment:
Could remove from the `optionalDeps` as well, please ?https://github.com/elastic/kibana/blob/1bf765def19c9d07e54f3bc7a322c28cbc8b6eb6/x-pack/plugins/features/kibana.json#L10

Reply:
done